src/main/scala/managehelpcontentpublisher/PathAndContent.scala

```diff
@@ -1,16 +1,59 @@
 package managehelpcontentpublisher
 
-import managehelpcontentpublisher.Article.fromInput
+import managehelpcontentpublisher.Article.{readArticle, writeArticle}
 import managehelpcontentpublisher.Config.config
 import managehelpcontentpublisher.Eithers._
-import upickle.default._
-
-import scala.util.Try
+import managehelpcontentpublisher.InputModel.readInput
+import managehelpcontentpublisher.MoreTopics.{readMoreTopics, writeMoreTopics}
+import managehelpcontentpublisher.Topic.{readTopic, writeTopic}
 
 case class PathAndContent(path: String, content: String)
 
 object PathAndContent {
 
+  private def removeFromTopics(
+      fetchTopicByPath: String => Either[Failure, Option[String]],
+      storeTopic: PathAndContent => Either[Failure, PathAndContent]
+  )(article: Article, topics: Seq[ArticleTopic]): Either[Failure, Seq[PathAndContent]] =
+    seqToEither(topics.map(removeFromTopic(fetchTopicByPath, storeTopic)(article))).map(_.flatten)
+
+  private def removeFromTopic(
+      fetchTopicByPath: String => Either[Failure, Option[String]],
+      storeTopic: PathAndContent => Either[Failure, PathAndContent]
+  )(article: Article)(articleTopic: ArticleTopic): Either[Failure, Option[PathAndContent]] =
+    for {
+      oldTopicJson <- fetchTopicByPath(articleTopic.path)
+      oldTopic <- optionToEither(oldTopicJson.map(readTopic))
```

Comment:
Seems you are inverting monads with `optionToEither` which should have exactly the same semantics as `sequence`. To aid familiarity you could provide `sequence` extension method and then `oldTopicJson.map(readTopic).sequence` might work